Homeworks/Homework W8/SQLQuery_HomeworkW8.sql

```diff
@@ -0,0 +1,340 @@
+CREATE DATABASE ZooManagemnt;
+
+USE ZooManagemnt;
+-- Creating tables
+CREATE TABLE ZoneArea
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Zone VARCHAR(50)
+);
+
+CREATE TABLE Species
```

Comment:
There should be consistency in the naming conventions you choose - some table names are singular (ZoneArea, Ticket) and other are plural (Species, Animals). Usually the plural form is used as inside a table we have multiple records for that specific entity. If you want to use singular you can but your approach should be consistent - all table names plural or all tables names singular.